src/components/views/Article/Article.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Tags from 'views/Tags';
+import './Article.scss';
+
+class Article extends Component {
+    render() {
+        const { id, date, title, author, tags, content } = this.props.post,
```

Comment:
@f3arnil Well, Article is really need `post` prop, so I think it's ok to throw error if this component are used without providing it with necessary `post` data.